Rakefile

```diff
@@ -203,13 +204,10 @@ task :jsondoc, :bundleupdate do |t, args|
 end
 
 namespace :jsondoc do
-  # @param tag [String] A valid tag, e.g. "v0.10.0" (legacy), "google-cloud-datastore:v1.2.0", or "master".
   desc "Clones gh-pages branch to a temp dir"
-  task :init, :tag do |t, args|
-    tag = args[:tag]
-    fail "Missing required parameter 'tag'." if tag.nil?
-    gh_pages = Pathname.new(Dir.home) + "tmp/#{tag}-gh-pages"
-
+  task :clone_gh_pages, [:gh_pages_dir] do |t, args|
```

Comment:
It might be a good idea to move helper tasks like `jsondoc: clone_gh_pages` and `jsondoc: copy` and `jsondoc: too` out into a separate object that can be unit tested, and keeping tasks only for the actions that can be instigated by a user or a process like travis/appveyor.

But that could be a separate refactor after the desired behavior is achieved.